integration-tests/frontend/test-class-wpseo-frontend-robots.php

```diff
@@ -193,6 +193,7 @@ public function test_category_noindex() {
 	public function test_author_archive() {
 		// Go to author page.
 		$user_id = $this->factory->user->create();
+		$this->factory->post->create_many( 1, array( 'post_author' => $user_id ) );
```

Comment:
I think you can also use the function `create` for creating 1 post, rather than using `create_many` and specify `1`. See [here](https://pippinsplugins.com/unit-tests-for-wordpress-plugins-the-factory/).